src/main/resources/com/linecorp/armeria/server/docs/assets/css/armeria.css

```diff
@@ -148,4 +157,15 @@ h2 {
 
 .debug-textarea {
   width: 100%;
+  max-width: 100%;
```

Comment:
Try to drag the resize bar at the right bottom of the text area, then you'll see the difference.